### ISSUE TITLE
0 and 1 inputs in database. Why?

### ISSUE THREAD

**Issue by duttanaman1** [TSonono/collaborative-filtering#13]
I had changed the array variable allowedRatings in the program and it shows the result.

Is that result obtained is correct?

**Comment by TSonono** [comment]
Hi,

As stated in the README, the ratings should currently follow the following format:


>a matrix containing of 0:s (not rated) and 1:s (liked the item) and follow this format.


No checks are currently done to ensure that the data in the rating matrix only consists of 0:s and 1:s, meaning you will get a result but a result that is meaningless (incorrect)

**Comment by duttanaman1** [comment]
For non-binary rating values, how should I proceed with the code? 
Any suggestions from your side. Like changing similarities functions, or changes in Comatrix.
Suggestions from you will be appericiated.

**Comment by TSonono** [comment]
This should definitely be doable, I just have not made all the necessary research myself on the mathematics of how to expand the current implementation to support such data input. If you want to use the same approach as in this library, I would suggest reading about [Jaccard index](https://en.wikipedia.org/wiki/Jaccard_index). I think understanding this part might be more difficult than implementing the code, at least it was in my experience when I wrote this library. As it was a long time ago I wrote this library, I don't remember much about Jaccard similarities and unfortunately, I currently don't have time to expand the library myself.

Good luck!
